test_scripts/Defects/5_0/1900_Media_app_does_not_get_AUDIBLE_if_it_is_successfully_resuming_during_active_embedded_navigation.lua

```diff
@@ -0,0 +1,91 @@
+-- Expected result:
+-- 1) SDL must set AUDIBLE audioStreamingState to this media app
+-- and send via OnHMIStatus together with resumed HMILevel (FULL or LIMITED).
+-- Actual result:
+-- SDL does not resume HMILevel and audioStreamingState.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local test = require("user_modules/dummy_connecttest")
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Local Functions ]]
+local function ignitionOff()
+    common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications",
+      { reason = "IGNITION_OFF" })
+    common.getMobileSession():ExpectNotification("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+    common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered",
+      { unexpectedDisconnect = true })
+    common.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLClose")
+    :Do(function()
+      StopSDL()
+    end)
+end
+
+local function cleanSessions()
+    for i = 1, common.getAppsCount() do
+      test.mobileSession[i] = nil
+    end
+    utils.wait()
+end
+
+local function embeddedNaviSource()
+    common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    {eventName = "EMBEDDED_NAVI", isActive = true})
+end
+
+local function checkResumingMediaApp()
+    common.getHMIConnection():ExpectRequest("BasicCommunication.ActivateApp",
+    {appID = common.getHMIAppId()})
+    :Do(function(_,data)
+        common.getHMIConnection():SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+    end)
+    common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    {eventName = "EMBEDDED_NAVI", isActive = false})
+
+    common.getMobileSession():ExpectNotification("OnHMIStatus")
+    :ValidIf(function(_, data)
+        if (data.payload.hmiLevel == "FULL" and
+        data.payload.audioStreamingState == "AUDIBLE") then
+            return true
+        end
+        return false, "SDL does not set required HMILevel and audioStreamingState"
+    end)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+runner.Step("IGNITION_OFF", ignitionOff)
+runner.Step("Close mobile connection", cleanSessions)
+runner.Step("Ignition On", common.start)
```

Comment:
@HSavynetska It's better to use app's unexpected disconnect instead of ignition cycle